app/src/main/cpp/HandMeshRenderer.h

```diff
@@ -0,0 +1,61 @@
+/* -*- Mode: C++; tab-width: 20; indent-tabs-mode: nil; c-basic-offset: 2 -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#pragma once
+
+#include "vrb/CreationContext.h"
+
+namespace crow {
+
+enum class HandMeshStrategy { None, Skinned, Spheres };
+
+struct Controller;
+
+class HandMeshRenderer;
+typedef std::shared_ptr<HandMeshRenderer> HandMeshRendererPtr;
+
+class HandMeshRenderer {
+protected:
+    vrb::CreationContextWeak context;
+    bool enabled = false;
+    virtual void Shutdown() { };
```

Comment:
It's unclear to me why we need a virtual protected Shutdown() method. The implementation is empty so no need for subclasses to call it. Also as far as I see there is no common code calling it so not needing to be virtual

Reply:
This is for derived classes to shutdown (Skinned uses it), but it probably should be called in the destructor of the base class. This is a common pattern in vrb classes: explicit Initialize(), Shutdown().

I will rework it a bit to simplify it, but I think we need some kind of Shutdown() method, and not all derived classes will need it.

Reply:
I don't think we need to follow any vrb pattern here, there are good reasons for Initialize/Shutdown in vrb but not here really where no external code is calling them

Reply:
I left the Shutdown() method virtual but called it from the destructor of the base class. This way, only subclasses that need a Shutdown() will override it, but they don't need to override the destructor, since the base  already calls Shutdown().

I will push this change and we can iterate it further.

Reply:
The thing is why do you need to add a Shutdown() when you already have the destructor? What does it add up?